runtime/centrifuge/src/migrations.rs

```diff
@@ -21,7 +21,7 @@ const IDENTITY_MIGRATION_KEY_LIMIT: u64 = 1000;
 
 parameter_types! {
 	// Address used by Anemoy to withdraw in AssetHub
-	// 4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2 --> 5Fc9NzKzJZwZvgjQBmSKtvZmJ5oP6B49DFC5dXZhTETjrSzo
+	// 4dTeMxuPJCK7zQGhFcgCivSJqBs9Wo2SuMSQeYCCuVJ9xrE2 --> 5CSfibAR9HHEwKpXbdsDHguAmLgKS6yKat2joTLA75DZ152L
```

Comment:
Luckily, only the comment was off. The hex is correct, see @lemunozm's original commit: https://github.com/centrifuge/centrifuge-chain/pull/1853/commits/f457f223ded12b8409250421a6faf5ff1a25312d#diff-99b28d05634bd6aa324ac36d08291a48d92664bbe2eef597f04ee84ec9e8de0aL56

I verified this using Chopsticks pre and post RU execution.

The reason the comment is off because `subkey inspect $hex` assumes the hex to be a private key, I should have used `subkey inspect $hex --public` instead:

```
❯ subkey inspect "0x10c03288a534d77418e3c19e745dfbc952423e179e1e3baa89e287092fc7802f" --public
Network ID/Version: substrate
  Public key (hex):   0x10c03288a534d77418e3c19e745dfbc952423e179e1e3baa89e287092fc7802f
  Account ID:         0x10c03288a534d77418e3c19e745dfbc952423e179e1e3baa89e287092fc7802f
  Public key (SS58):  5CSfibAR9HHEwKpXbdsDHguAmLgKS6yKat2joTLA75DZ152L
  SS58 Address:       5CSfibAR9HHEwKpXbdsDHguAmLgKS6yKat2joTLA75DZ152L
❯ subkey inspect 5Fc9NzKzJZwZvgjQBmSKtvZmJ5oP6B49DFC5dXZhTETjrSzo
Public Key URI `5Fc9NzKzJZwZvgjQBmSKtvZmJ5oP6B49DFC5dXZhTETjrSzo` is account:
  Network ID/Version: substrate
  Public key (hex):   0x9caf39a5c322253288007349ea052d49d16f5344a94c4879d9e185404fb93851
  Account ID:         0x9caf39a5c322253288007349ea052d49d16f5344a94c4879d9e185404fb93851
  Public key (SS58):  5Fc9NzKzJZwZvgjQBmSKtvZmJ5oP6B49DFC5dXZhTETjrSzo
  SS58 Address:       5Fc9NzKzJZwZvgjQBmSKtvZmJ5oP6B49DFC5dXZhTETjrSzo
```

Reply:
Thanks for the comment!